models/complete_order.py

```diff
@@ -1,46 +1,96 @@
+import sqlite3
+
 class Order():
-    def __init__(self, customer, payment_type):
+    def __init__(self):
 
         """
         Stores a single instance of relationship between
         :model:`customer`, :model:`payment_type`
-
         author: Whitney Cormack, Temporary Oysters
-        # customer, payment type, order status
+        # customer, payment type
         """
-        self.customer = customer
-        self.payment_type = payment_type
-        self.order_status = False
-
+        self.customer = self.retrieve_active_customer_from_db();
+        self.payment_type = self.retrieve_payment_type_from_db();
 
     def get_customer(self):
-        """
-        returns customer information
-        """
+        """ returns active customer on order """
         return self.customer
 
     def get_payment_type(self):
-        """
-        returns payment information
-        """
+        """ returns active customer's payment on order """
         return self.payment_type
 
-    def get_order_status(self):
-        """
-        returns order status
-        False = not a complete order
-        """
-        return self.order_status
+    def retrieve_active_customer_from_db(self):
+        """ returns active customer on order which will be set to self.customer """
+        with sqlite3.connect('../bangazon.db') as roncon:
+            cursor = roncon.cursor()
 
-    def order_status_is_complete(self):
-        """
-        set order status
-        True = complete order
+            cursor.execute("""
+                SELECT * FROM Customer
+                WHERE status = 1
+            """)
+            selected_customer = cursor.fetchone()
+            # print("$$$$$$$$$$$$ selected_customer", selected_customer)
+            return selected_customer[0]
+
+    def retrieve_payment_type_from_db(self):
+        """ returns customer's payment type on order """
+        with sqlite3.connect('../bangazon.db') as roncon:
+            cursor = roncon.cursor()
+
+            cursor.execute("""
+                SELECT * FROM PaymentOption
+                WHERE customer='{}'
+            """.format(self.get_customer()
+                ))
+            selected_payment_option = cursor.fetchall()
+            # print("****** Selected_payment_option:", selected_payment_option, self.get_customer())
+            if len(selected_payment_option) > 0:
+                return selected_payment_option[0][0]
+            else:
+                return False
```

Comment:
Seems to be returning False as displayed in the database. Check DB Browser on your machine and see if it's the same.